test/anyP.js

```diff
@@ -0,0 +1,191 @@
+import * as R from 'ramda';
+import { assert } from 'chai';
+
+import * as RA from '../src';
+import { anyPPolyfill, AggregatedError } from '../src/anyP';
+
+describe('anyP', function() {
+  context('given list of fulfilled promises', function() {
+    specify('should return first of fulfilled values', async function() {
+      const p1 = RA.resolveP(1);
+      const p2 = RA.resolveP(2);
+      const p3 = RA.resolveP(3);
+      const actual = [p1, p2, p3];
+      const expected = 1;
+
+      assert.strictEqual(await RA.anyP(actual), expected);
+    });
+  });
+
+  context('given list of rejected promises', function() {
+    specify(
+      'should fail with AggregatedError that contains a list of rejected reasons',
+      async function() {
+        const p1 = RA.rejectP(1);
+        const p2 = RA.rejectP(2);
+        const p3 = RA.rejectP(3);
+        const actual = [p1, p2, p3];
+        const expected = [1, 2, 3];
+
+        try {
+          await RA.anyP(actual);
+          throw new Error('resolving should fail');
+        } catch (e) {
+          assert.notStrictEqual(e.message, 'resolving should fail');
+          assert.strictEqual(e instanceof AggregatedError, true);
+          assert.deepEqual(e.exceptions, expected);
+        }
+      }
+    );
+  });
+
+  context('given list of fulfilled and rejected promises', function() {
+    specify('should return first of fulfilled values', async function() {
+      const p1 = RA.rejectP(1);
+      const p2 = RA.resolveP(2);
+      const p3 = RA.rejectP(3);
+      const actual = [p1, p2, p3];
+      const expected = 2;
+
+      assert.strictEqual(await RA.anyP(actual), expected);
+    });
+  });
+
+  context('given list of values', function() {
+    specify('should return first of fulfilled values', async function() {
+      const v1 = 1;
+      const v2 = 2;
+      const v3 = 3;
+      const actual = [v1, v2, v3];
+      const expected = 1;
+
+      assert.strictEqual(await RA.anyP(actual), expected);
+    });
+  });
+
+  context('given list of values and promises', function() {
+    specify('should return first of fulfilled values', async function() {
```

Comment:
This test doesn't correspond with example here: https://github.com/char0n/ramda-adjunct/pull/1080/files#diff-4cb076891bd6a121df6d7ea58b2d06b7R27

Reply:
Thank you, I fixed it